x-pack/plugins/security_solution/public/resolver/store/data/graphing.test.ts

```diff
@@ -187,3 +204,48 @@ describe('resolver graph layout', () => {
     });
   });
 });
+
+describe('resolver graph with too much lineage', () => {
+  let generator: EndpointDocGenerator;
+  let store: Store<DataState, DataAction>;
+  let allEvents: ResolverEvent[];
+  let childrenCursor: string;
+  let ancestorCursor: string;
+
+  beforeEach(() => {
+    generator = new EndpointDocGenerator('seed');
+    allEvents = generator.generateTree({ ancestors: 1, generations: 2, children: 2 }).allEvents;
+    childrenCursor = 'aValidChildursor';
+    ancestorCursor = 'aValidAncestorCursor';
+    store = createStore(dataReducer, undefined);
+  });
+
+  describe('should select from state properly', () => {
+    it('has the correct ancestor cursor', () => {
```

Comment:
I expected to see something like 'should indicate that there is too much data'.

Reply:
changed it: https://github.com/elastic/kibana/pull/69590/commits/5c6a45349adfc523dc8c946adb7eda70fa4ea7cb